<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindowForm</class>
 <widget class="QWidget" name="TestWindowForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="questionNumberLabel">
       <property name="text">
        <string>QuestionNumber</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>813</width>
          <height>291</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="questionLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="text">
            <string>Question</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="answerListView">
       <property name="editTriggers">
        <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="AnswerVerticalLayout">
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sourceLabel">
         <property name="text">
          <string>source</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="tagsPushButton">
         <property name="text">
          <string>Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="checkButton">
         <property name="text">
          <string>Check</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>checkButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>771</x>
     <y>680</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked()</signal>
   <receiver>TestWindowForm</receiver>
   <slot>CheckAnswers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>798</x>
     <y>676</y>
    </hint>
    <hint type="destinationlabel">
     <x>833</x>
     <y>649</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked()</signal>
   <receiver>answerListView</receiver>
   <slot>doItemsLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>699</y>
    </hint>
    <hint type="destinationlabel">
     <x>704</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>sourceLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>682</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>653</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>descriptionLabel</receiver>
   <slot>setHidden(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>686</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableNextButton()</slot>
  <slot>CheckAnswers()</slot>
  <slot>Back()</slot>
  <slot>SignalFromCheckButton()</slot>
 </slots>
</ui>
